src/styles/index.scss

```diff
@@ -1,3 +1,231 @@
-body {
+// body {
+//   margin: 0;
+//   font-family: Roboto, sans-serif;
+// }
+
+// .card {
+//   max-width: 200px;
+// }
+
+// .card__container {
+//   // box-sizing: border-box;
+//   display: flex;
+//   flex-direction: column;
+//   border: 1px solid #f3f3f3;
+//   border-radius: 5px;
+//   max-width: 166px;
+//   margin: 0;
+//   align-items: center;
+//   padding: 32px 16px 16px;
+// }
+
+// .card__img {
+//   display: block;
+//   width: 160px;
+//   height: 134px;
+//   margin: 0;
+// }
+
+// .card__title {
+//   font-weight: 500;
+//   font-size: 12px;
+//   line-height: 18px;
+//   color: #060b35;
+
+//   margin-top: 40px;
+//   margin-bottom: 4px;
+// }
+
+// .card__desc {
+//   width: 100%;
+//   font-weight: 400;
+//   font-size: 10px;
+//   line-height: 14px;
+//   color: #616070;
+
+//   margin-bottom: 16px;
+// }
+
+// .card__review {
+//   width: 100%;
+//   display: flex;
+//   justify-content: space-between;
+
+//   margin-bottom: 24px;
+// }
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('./../images/star.svg');
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url('./../images/star-active.svg');
+}
+
+// .card__review-text {
+//   align-self: flex-end;
+//   font-weight: 400;
+//   font-size: 10px;
+//   line-height: 14px;
+//   color: #060b35;
+// }
+
+// .card__price {
+//   width: 100%;
+//   display: flex;
+//   justify-content: space-between;
+//   align-items: center;
+
+//   margin-bottom: 16px;
+// }
+
+// .card__price-text {
+//   font-weight: 400;
+//   font-size: 12px;
+//   line-height: 18px;
+//   color: #616070;
+// }
+
+// .card__price-value {
+//   font-weight: 700;
+//   font-size: 16px;
+//   line-height: 18px;
+//   color: #060b35;
+// }
+
+// .card__btn {
+//   display: flex;
+//   align-items: center;
+//   justify-content: center;
+//   box-sizing: border-box;
+//   width: 100%;
+//   height: 40px;
+//   background: #00acdc;
+//   border: 1px solid transparent;
+//   border-radius: 5px;
+//   color: #fff;
+//   font-weight: 700;
+//   font-size: 14px;
+//   line-height: 1.14;
+//   text-transform: uppercase;
+//   transition:
+//     background-color 0.3s,
+//     color 0.3s;
+// }
+
+// .card__btn:hover {
+//   background-color: #fff;
+//   border: 1px solid #00acdc;
+//   color: #00acdc;}
+
+// // ...................
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: #060b35;
+}
```

Comment:
Font family and other font-related properties that are reused throughout the file should be extracted into a variable or mixin for better consistency and easier maintenance.